pom.xml

```diff
@@ -162,7 +163,11 @@
             <groupId>org.springdoc</groupId>
             <artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
         </dependency>
-
+        <dependency>
```

Comment:
Add a todo to remove this dependency when it will be added in gridsuite dependencies